### PR TITLE
Documentation : Added Additional Description to PhysicsServer2D->area_create() 

### LINKED ISSUE #46696: Doc for Physics2DServer.area_create() should mention that the area have to be added to world
**Godot version:**

4de0768 / 3.2.4 rc

**Issue description:**

I was using Physics2DSever to optimise my collisions by creating areas directly from the server and encountered an issue where my areas didn't detect anything. As it turns out, you need to add the area to a physics space to have it work.

The Physics2DServer documentation is generally lacking, but the `area_create()` method description should mention that you have to call `area_set_space()` on that area to make it work properly, so that people using it won't waste time figuring out why it's broken (like me for the last hour ;_;).

### PR DESCRIPTION
**Pull request by Bhu1-V**
Tries to improve PhysicsServer2D doc for issue #46696

for PR

*Bugsquad edit:* Fixes #46696.

### CODE REVIEW

File: doc/classes/PhysicsServer2D.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
 			<return type="RID">
 			</return>
 			<description>
-				Creates an [Area2D].
+				Creates an [Area2D]. After creating the area with this method assign it with a space using [method area_set_space] to use the created Area.
```
Comment (Calinou) [inline doc/classes/PhysicsServer2D.xml:61]:
```suggestion
				Creates an [Area2D]. After creating an [Area2D] with this method, assign it with a space using [method area_set_space] to use the created [Area2D].
```

**Review by Calinou — DISMISSED** [review]

File: doc/classes/PhysicsServer2D.xml (hunk 1 above)
Comment (Bhu1-V) [reply doc/classes/PhysicsServer2D.xml]: Sure.

File: doc/classes/PhysicsServer2D.xml [hunk 2]
```diff
@@ -58,7 +58,7 @@
 			<return type="RID">
 			</return>
 			<description>
-				Creates an [Area2D].
+				Creates an [Area2D]. After creating an [Area2D] with this method, assign it with a space using [method area_set_space] to use the created [Area2D].
```
Comment (pouleyKetchoupp) [inline doc/classes/PhysicsServer2D.xml:61]: I would suggest this change to describe the purpose a bit more clearly:
```suggestion
				Creates an [Area2D]. After creating an [Area2D] with this method, assign it to a space using [method area_set_space] to use the created [Area2D] in the physics world.
```

**Review by pouleyKetchoupp — CHANGES_REQUESTED** [review]

File: doc/classes/PhysicsServer2D.xml (hunk 2 above)
Comment (Bhu1-V) [reply doc/classes/PhysicsServer2D.xml]: Sure.

File: doc/classes/PhysicsServer2D.xml [hunk 3]
```diff
@@ -58,7 +58,7 @@
 			<return type="RID">
 			</return>
 			<description>
-				Creates an [Area2D].
+				Creates an [Area2D]. After creating an [Area2D] with this method, assign it with a space using [method area_set_space] to use the created [Area2D] in the physics world.
```
Comment (pouleyKetchoupp) [inline doc/classes/PhysicsServer2D.xml:61]: It still says `assign it with a space` instead of `assign it to a space`.

Comment (Bhu1-V) [reply doc/classes/PhysicsServer2D.xml]: sure

**Review by pouleyKetchoupp — APPROVED** [review]
Looks good!

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**